gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/orc/HiveOrcSerDeManager.java

```diff
@@ -152,18 +147,12 @@ public void addSerDeProperties(HiveRegistrationUnit source, HiveRegistrationUnit
     if (source.getOutputFormat().isPresent()) {
       target.setOutputFormat(source.getOutputFormat().get());
     }
-    if (source.getSerDeProps().contains(SCHEMA_LITERAL)) {
-      target.setSerDeProp(SCHEMA_LITERAL, source.getSerDeProps().getProp(SCHEMA_LITERAL));
-    }
   }
 
   @Override
   public void updateSchema(HiveRegistrationUnit existingUnit, HiveRegistrationUnit newUnit)
       throws IOException {
-    Preconditions.checkArgument(
-        newUnit.getSerDeProps().contains(SCHEMA_LITERAL));
-
-    existingUnit.setSerDeProp(SCHEMA_LITERAL, newUnit.getSerDeProps().getProp(SCHEMA_LITERAL));
+    log.debug("Do nothing to update schema since orc.schema.literal is removed");
```

Comment:
Getting rid of this method override

Reply:
Do we want to remove this method? Or we do the update on columns and column.types? Since I'm not sure whether this method is called in other projects

Reply:
I thought logging is the only thing that you did in the method, if that is the case, there's no need to keep this overriding. 
But yes updating `columns` and `column.types` should be the correct behavior. It actually doesn't matter too much if this method is called anywhere because it is a public method and there could be usage somewhere we have no way to control, but still we strive to maintain backward-compatability with correct behavior.